clients/nodejs/CorePromiseClient.js

```diff
@@ -33,6 +46,10 @@ class CorePromiseClient {
     this.client.getLastUserStateTransitionHash = promisify(
       this.client.getLastUserStateTransitionHash.bind(this.client),
     );
+
+    this.client.subscribeToBlockHeadersWithChainLocks = promisify(
```

Comment:
Stream methods don’t accept callbacks. They return event emitter. So you don’t need to promisify them.